### PR TITLE
Update HTTP spec-urls frontmatter keys to current HTTP spec

### PR DESCRIPTION
**Pull request by sideshowbarker**
After spending some time thinking about it more, I figured out some ways to automate the updates for most of these.

### CODE REVIEW

File: files/en-us/web/http/status/429/index.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ tags:
   - HTTP Status Code
   - Reference
   - Status code
-spec-urls: https://www.rfc-editor.org/rfc/rfc6585#section-4
+spec-urls: https://httpwg.org/specs/rfc9110.html#status.429
```
Comment (teoli2003) [inline files/en-us/web/http/status/429/index.md:10]: 429 is not defined in this specification.

File: files/en-us/web/http/status/511/index.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ tags:
   - Reference
   - Server error
   - Status code
-spec-urls: https://www.rfc-editor.org/rfc/rfc6585#section-6
+spec-urls: https://httpwg.org/specs/rfc9110.html#status.511
```
Comment (teoli2003) [inline files/en-us/web/http/status/511/index.md:10]: 511 is not defined in this specification

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/web/http/status/429/index.md (hunk 1 above)
Comment (sideshowbarker) [reply files/en-us/web/http/status/429/index.md]: oops — pushed an update which reverts that

File: files/en-us/web/http/status/511/index.md (hunk 2 above)
Comment (sideshowbarker) [reply files/en-us/web/http/status/511/index.md]: thanks for catching — reverted in update

**Review by teoli2003 — APPROVED** [review]
👍

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**